snuba/datasets/configuration/discover/entities/discover.yaml

```diff
@@ -363,8 +363,8 @@ storages:
         - mapper: DefaultNoneSubscriptMapper
           args:
             subscript_names:
-              - measurements
               - span_op_breakdowns
+              - measurements
```

Comment:
Why this change?

Reply:
It's an artifact of me changing the order in the discover.py file first and then seeing that it caused a difference in this file. More operations than necessary but no impact either way